web/satellite/src/utils/stripe.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export default class Stripe {
+    private cardElement: any;
+
+    private stripe: any;
+
+    private createCardCallback: (result: any) => any;
+
+    public newCardInput(resultCallback: (result: any) => any) {
+        this.createCardCallback = resultCallback;
+
+        if (!window['Stripe']) {
+            throw new Error('Stripe library not loaded');
+        }
+
+        this.stripe = window['Stripe'](process.env.VUE_APP_STRIPE_PUBLIC_KEY);
+        if (!this.stripe) {
+            throw new Error('Unable to initialize stripe');
+        }
+
+        const elements = this.stripe.elements();
+        if (!elements) {
+            throw new Error('Unable to instantiate elements');
+        }
+
+        this.cardElement = elements.create('card');
+        if (!this.cardElement) {
+            throw new Error('Unable to create card');
+        }
+
+        this.cardElement.mount('#card-element');
+
+        this.cardElement.addEventListener('change', function (event) {
+            const displayError = document.getElementById('card-errors') as HTMLElement;
```

Comment:
Is it possible to refactor it like that
 const displayError: HTMLElement = document.getElementById('card-errors'); ?